# HG changeset patch
# User Alan Robertson <alanr@unix.sh>
# Date 1185658580 21600
# Node ID 3ad95813d17f3d1adf0c38713ee09dca5e45cd5d
# Parent  f5f06cad7734ccfaef80ee98068fef7230879372
Patch to revert configure.in check from Tuomo Soini - LF bug 1660

--- 
31st July 2007: Reverse this patch as it breaks the build for ia64 Debian.
                Horms


diff -r f5f06cad7734 -r 3ad95813d17f configure.in
--- a/configure.in	Fri Jul 27 10:16:24 2007 -0600
+++ b/configure.in	Sat Jul 28 15:36:20 2007 -0600
@@ -1032,7 +1032,7 @@ CPPFLAGS="$CPPFLAGS $XML2HEAD"
 
 dnl ************************************************************************
 dnl * Check for linux/icmpv6.h to make enable the IPv6addr resource agent
-AC_CHECK_HEADERS(linux/icmpv6.h,[],[],[#include <asm/types.h>])
+AC_CHECK_HEADERS(linux/icmpv6.h,[],[],[#include <sys/types.h>])
 AM_CONDITIONAL(USE_IPV6ADDR, test "$ac_cv_header_linux_icmpv6_h" = yes -a $new_libnet = yes )
 AC_CHECK_HEADERS(libxml/xpath.h)
 
